docker-storage-setup.sh

```diff
@@ -865,6 +969,16 @@ setup_storage() {
    Fatal "Storage is already configured with ${current_driver} driver. Can't configure it with ${STORAGE_DRIVER} driver. To override, remove $DOCKER_STORAGE and retry."
   fi
 
+  if [ "$STORAGE_DRIVER" == "devicemapper" ] && [ "$DOCKER_ROOT_VOLUME" == "yes" ];then
+   Fatal "STORAGE_DRIVER=${STORAGE_DRIVER} and DOCKER_ROOT_VOLUME=${DOCKER_ROOT_VOLUME} are mutually exclusive options. Use either one of them, not both."
+  fi
```

Comment:
shishir, I am changing my mind on this now. There is no real technical reason that why root volume can't be allowed with devicemapper storage driver. So lets get rid of this restriction.